sdk/core/core-amqp/src/errors.ts

```diff
@@ -560,6 +553,19 @@ export function translate(err: AmqpError | Error): MessagingError {
 
   let error: MessagingError = err as MessagingError;
 
+  // OperationTimeoutError is retryable as it indicates potential problem with unexpected delays
+  // and not an issue with the Messaging process
```

Comment:
> and not an issue with the Messaging process

Am guessing this was copied over from the next if block :) ?

We should remove this line. OperationTimeoutError is when the service does not return back to us with an acknowledgment of some sorts and therefore related to the Messaging process

Reply:
No this was based on understanding of scenarios when OperationTimeoutError maybe throw, which can be delays

> when the service does not return back to us with an acknowledgment of some sorts

About above, that seems to be a general definition of an error isn't it? What do you propose using the description for `OperationTimeoutError` (that would describe it more than an error description would) ?

Reply:
> What do you propose using the description for OperationTimeoutError

This part of the code does not require any error description. `OperationTimeoutError` is a very self explanatory error. My concern was about making the statement that "OperationTimeoutError is not an issue with the Messaging process" which is not true

At most what we can add here is, `OperationTimeoutError occurs when the service fails to respond withing a given timeframe. Since reasons for such failures can be transient, this is a retryable error.`
The only place where we would want to describe `OperationTimeoutError` is the place where it is defined. In our case, that is in rhea-promise which describes the error just fine. https://github.com/amqp/rhea-promise/blob/3fdbbd53fe8c69896b03b46f5c12a93e7c8e119d/lib/errorDefinitions.ts#L5